content/dips/DIP-18.md

```diff
@@ -0,0 +1,88 @@
+---
+DIP: 18
+title: dYdX Operations subDAO
+status: Proposed
+author: Reverie
+shortDescription: Establish a Guernsey Purpose Trust with $360,000
+discussions: https://commonwealth.im/dydx/discussion/7620-drc-dydx-operations-trust
+created: 2022-11-22
+---
+
+## Simple Summary
+
+Launch a dYdX Operations subDAO by establishing a Guernsey Purpose Trust entity funded with $360,000 of DYDX transferred from the Community Treasury.
+
+## Abstract
+
+Reverie is proposing to launch an Operations subDAO for the dYdX DAO to manage the publication of a DAO Playbook, run DAO communications, open a fiat bank account, and additional responsibilities.
+
+The subDAO will be established through a Guernsey Purpose Trust, named the dYdX Operations Trust. Three Trustees and an Enforcer will be appointed to manage the responsibilities of the Trust.
+
+To cover compensation and expenses, we are requesting a total of $360,000 in DYDX to be transferred from the Community Treasury to the Ops Trust multisig.
+The Operations subDAO will be operational for an initial 6 month period to deliver the responsibilites outlined below.
+
+The snapshot vote concluded with X voters and XM DYDX in agreement (X%). The vote results have shown significant support for the program.
+https://snapshot.org/#/dydxgov.eth/proposal/0xa710b0ca85e04ac2db123ae522ac5030a008ccbc8d4d413e83d18f6032b39758
+
+**Relevant Links**
+
+DRC discussion: https://commonwealth.im/dydx/discussion/7620-drc-dydx-operations-trust
+
+## Motivation
+
+dYdX is facing an important decision to scale the protocol thoughtfully as it transitions to full community ownership. As a community, we have to look ahead towards organizational transformation through the use of subDAOs. Accomplishing this goal will first require guidance on structural implementation.
```

Comment:
"dYdX is facing an important decision to scale the protocol thoughtfully as it transitions to full community **control**."